mgmtsystem/ec2.py

```diff
@@ -467,3 +493,30 @@ def release_address(self, address):
     def get_all_unattached_volumes(self):
         return [volume for volume in self.api.get_all_volumes() if not
                 volume.attach_data.status]
+
+    def delete_sqs_queue(self, queue_id):
+        queue_name = "manageiq-awsconfig-queue-{}".format(queue_id)
```

Comment:
CloudForms is using manageiq-awsconfig-queue string for naming SQS queue aswell.